nimare/tests/test_transforms.py

```diff
@@ -247,18 +247,22 @@ def test_ddimages_to_coordinates_merge_strategy(testdata_ibma):
 
 
 @pytest.mark.parametrize(
-    "z,tail,expected_p",
+    "expected_z,tail,expected_p",
     [
         (0.0, "two", 1.0),
         (0.0, "one", 0.5),
         (1.959963, "two", 0.05),
         (1.959963, "one", 0.025),
         (-1.959963, "two", 0.05),
+        (-1.959963, "one", 0.025),
```

Comment:
This test case should be (-1.959964, "one", 0.0975). I think I've corrected in issue #748